packages/toolkit/src/components/top-bar/NavLinks.tsx

```diff
@@ -128,19 +134,23 @@ export const NavLinks = ({ isExploreRoute }: { isExploreRoute?: boolean }) => {
     accessToken,
   });
 
+  const isCloud = env("NEXT_PUBLIC_APP_ENV") === "CLOUD";
+
   return (
     <React.Fragment>
       {me.isSuccess
-        ? navLinkItems.map(({ pathname, Icon, title }) => (
-            <NavLink
-              key={pathname}
-              pathname={pathname}
-              Icon={Icon}
-              title={title}
-              isExploreRoute={isExploreRoute}
-              strict={true}
-            />
-          ))
+        ? navLinkItems
+            .filter((item) => isCloud || item.pathname !== "applications")
```

Comment:
@thewbuk This logic is kind of hard to read, let's don't use one-liner here, instead

if (isCloud){
  return true
}

return item.pathname !== "application"